Makefile

```diff
@@ -21,6 +21,7 @@ LD_FLAGS                    := "-w -X github.com/gardener/gardener-extensions/pk
 VERIFY                      := true
 LEADER_ELECTION             := false
 IGNORE_OPERATION_ANNOTATION := false
+CERTIFICATE_SERVICE_CONFIG  := ./controllers/extension-certificate-service/example/service-config.yaml
```

Comment:
I would call the file just "config.yaml" (when I first read I was confused what "service config" might be, only the `CERTIFICATE_SERVICE_CONFIG` gave me a hint ;-)).